test/integration/formatting_stack/strategies.clj

```diff
@@ -1,28 +1,53 @@
 (ns integration.formatting-stack.strategies
   (:require
-   [clojure.java.shell :refer [sh]]
+   [clojure.java.shell :as shell]
    [clojure.set :as set]
    [clojure.string :as string]
-   [clojure.test :refer [deftest is testing]]
+   [clojure.test :refer [deftest is testing use-fixtures]]
    [formatting-stack.strategies :as sut]
+   [formatting-stack.strategies.impl :as impl]
+   [formatting-stack.test-helpers :as test-helpers]
    [formatting-stack.util :refer [rcomp]]
    [nedap.speced.def :as speced])
   (:import
    (java.io File)))
 
-(def ^File deletable-file (File. "test-resources/deletable.clj"))
+;;; move to test helpers?
+(defn sh [& args]
+  (apply shell/sh (conj (vec args) :dir impl/*pwd*)))
 
-(def ^File rename-destination (File. "test-resources/deletable_moved.clj"))
+(def ^File deletable-file (File. "tmp/test-repo/test-resources/deletable.clj"))
 
-(def ^String createable-filename "test-resources/createable.clj")
+(def ^File rename-destination (File. "tmp/test-repo/test-resources/deletable_moved.clj"))
+
+(def ^String creatable-filename "tmp/test-repo/test-resources/creatable.clj")
 
 (def creatable-contents (pr-str '(ns foo)))
 
-(assert (-> deletable-file .exists))
+(defn cleanup-repo []
```

Comment:
maybe can be named `cleanup-testing-repo!` so that it's clearer that the main repo is not intended to be cleaned up